projectgenerator/gui/generate_project.py

```diff
@@ -538,6 +544,12 @@ def fill_models_line_edit(self):
         self.ili_models_line_edit.setText(
             self.multiple_models_dialog.get_models_string())
 
+    def fill_toml_file_info_label(self):
+        text = None
+        if self.ili2db_options.toml_file():
+            text = 'Ext. Attributes File: '+('...'+self.ili2db_options.toml_file()[len(self.ili2db_options.toml_file())-60:]) if len(self.ili2db_options.toml_file()) > 60 else 'Ext. Attributes File: '+self.ili2db_options.toml_file()
```

Comment:
text = `self.tr('Ext. Attributes File: {}').format(self.ili2db_options.toml_file())` or similar... And for the ellipsis use `…` ;)